.github/workflows/kvrocks.yaml

```diff
@@ -426,6 +426,9 @@ jobs:
           - name: ArchLinux
             image: archlinux:base
             compiler: gcc
+          - name: Debian 12
+            image: debian:bookworm-slim
```

Comment:
Maybe use a base image: bookworm, not a slim version

Reply:
I think it's fine to use the same image as our docker image to cover all cases for any problem in the docker image.

Reply:
👍

Reply:
Ok, make sense